### PR TITLE
Complete inlining and remove DASK stubs out

### LINKED ISSUE #152: Inline method docs, rather than using "derived_from"
The problem is that the equivalent methods in pandas usually have a lot of arguments, and some of them don't make to support in Koalas or are temporarily not supported. Showing those in the examples are simply confusing to end users when they read the docs. We should just inline the docs. We can however create them using pandas docs as the baseline.

Here is an initial list:

```
frame.py-    def iteritems(self):
frame.py-    def to_html(self, buf=None, columns=None, col_space=None, header=True, index=True,
frame.py-    def isnull(self):
frame.py-    def notnull(self):
frame.py-    def toPandas(self):
frame.py-    def assign(self, **kwargs):
frame.py-    def dropna(self, axis=0, how='any', thresh=None, subset=None, inplace=False):
frame.py-    def drop(self, labels, axis=0, errors='raise'):
frame.py-    def get(self, key, default=None):
frame.py-    def pipe(self, func, *args, **kwargs):

series.py-    def rename(self, index=None, **kwargs):
series.py-    def reset_index(self, level=None, drop=False, name=None, inplace=False):
series.py-    def isnull(self):
series.py-    def notnull(self):
series.py-    def dropna(self, axis=0, inplace=False, **kwargs):
series.py-    def head(self, n=5):
series.py-    def value_counts(self, normalize=False, sort=True, ascending=False, bins=None, dropna=True):
```

### PR DESCRIPTION
**Pull request by HyukjinKwon**
This PR proposes two things:

- Get rid of DASK things out in Koalas
- Complete inlining docs

Resolves #152

### CODE REVIEW

File: databricks/koalas/frame.py [hunk 1]
```diff
@@ -47,9 +45,73 @@ class DataFrame(_Frame):
...
+        Data type to force. Only a single dtype is allowed. If None, infer
+    copy : boolean, default False
+        Copy data from inputs. Only affects DataFrame / 2d ndarray input
+
+    Examples
+    --------
+    Constructing DataFrame from a dictionary.
+
+    >>> d = {'col1': [1, 2], 'col2': [3, 4]}
+    >>> df = ks.DataFrame(data=d)
```
Comment (ueshin) [inline databricks/koalas/frame.py]: This could be flaky with Python 3.5.

File: databricks/koalas/frame.py [hunk 2]
```diff
@@ -47,9 +45,73 @@ class DataFrame(_Frame):
...
+    >>> df
+       col1  col2
+    0     1     3
+    1     2     4
+
+    Constructing DataFrame from Pandas DataFrame
+
+    >>> df = ks.DataFrame(pd.DataFrame(data=d))
```
Comment (ueshin) [inline databricks/koalas/frame.py]: ditto.

**Comment by HyukjinKwon** [comment]
Let me get this in first but please let me know if there are more things to be done.

File: databricks/koalas/frame.py [hunk 3]
```diff
@@ -73,15 +73,15 @@ class DataFrame(_Frame):
...
+    >>> df = ks.DataFrame(data=d, columns=['col1', 'col2'])
     >>> df
        col1  col2
     0     1     3
     1     2     4
 
     Constructing DataFrame from Pandas DataFrame
 
-    >>> df = ks.DataFrame(pd.DataFrame(data=d))
+    >>> df = ks.DataFrame(pd.DataFrame(data=d), columns=['col1', 'col2'])
```
Comment (ueshin) [inline databricks/koalas/frame.py]: `ks.DataFrame(pd.DataFrame(data=d, columns=['col1', 'col2']))`?

Comment (HyukjinKwon) [reply databricks/koalas/frame.py]: Hm, let me address them with adding some assertions.

**Pull request merged**